config/samples/gatling-operator_v1alpha1_gatling01.yaml

```diff
@@ -42,12 +47,39 @@ spec:
     #      secretKeyRef:
     #        name: aws-credentail-secrets
     #        key: AWS_SECRET_ACCESS_KEY
+    #
+    #############################################################
+    #        Storage Provider - gcp (Google Cloud Storage)
+    #############################################################
+    # provider: "gcp"
+    # bucket: "gatling-operator-reports"                  # GCS bucket name on which Gatlilng report files are stored
+    # bucket: "zozo-mlops-dev-gatling-operator-reports"   # レポート用のGCSバケット zozo-mlops-dev(stg)-gatling-operator-reports を定義しているのでそちらを指定してください
```

Comment:
nits: I think this line is unnecessary.

Reply:
thanks. removed https://github.com/st-tech/gatling-operator/commit/c49c5091eca5c3ca5a0f5e77d17667e2c98acf21